style.css

```diff
@@ -0,0 +1,66 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+}
+
+.greetings {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.socials {
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  list-style-type: none;
+  margin-top: 14px;
+  padding-bottom: 2px;
+}
+
+.button {
+  color: #172b4d;
+  justify-content: center;
+  display: flex;
+  padding-bottom: 10px;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 20px;
+  text-align: center;
+  background: url("images/illustrator-mobile.svg") no-repeat center;
+  background-size: cover;
+  overflow-y: auto;
+}
```

Comment:
- [ ] Kindly make sure to add this background image on the whole `section` because if you notice this background image doesn't cover even the `button` while on the Figma design even the `button` is covered with this background image.

_**As a suggestion in order to fit your background image correctly as it is the Figma design give these styles to your `section` container instead of the `headline` container.**_

         background-image: url("images/illustrator-mobile.svg");
         background-size: 100% 100%;
         background-repeat: no-repeat;


  - ### YOUR DESIGN
  
    ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/87197412/73dbbf77-8c8a-4639-b7e2-4b382815847b)

  - ### FIGMA DESIGN
  
    ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/87197412/89150b9f-2ff7-4bb3-8a98-eccc984f741d)